### PR TITLE
session1

### PR DESCRIPTION
**Pull request by Neuvax**

### CODE REVIEW

**Review by Neuvax — COMMENTED** [review]
### Why
Practice create PR 

### What
Create a new PR request and get somebody on my team to review

### How
Did a small code change 
Push the branch to github

File: session1.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- counting records on customers table
+--
+SELECT count(*) from sh.CUSTOMERS;
+
+--
+--
```
Comment (Armandotrsg) [inline session1.sql]: What is this? Fix

**Review by Armandotrsg — CHANGES_REQUESTED** [review]
Fix

**Review by Armandotrsg — APPROVED** [review]
LGTM!

**Pull request merged**